gpflow/likelihoods.py

```diff
@@ -583,3 +583,136 @@ def conditional_variance(self, F):
         E_y = tf.matmul(phi, Ys)
         E_y2 = tf.matmul(phi, tf.square(Ys))
         return tf.reshape(E_y2 - tf.square(E_y), tf.shape(F))
+
+
+class MonteCarloLikelihood(Likelihood):
+    def __init__(self, name=None):
+        super().__init__(name)
+        self.num_monte_carlo_points = 100
+        del self.num_gauss_hermite_points
+
+    def _mc_evals(self, func, Fmu, Fvar, Y=None, epsilon=None):
```

Comment:
Should this not be a static method or just a function somewhere? I don't see it referring to `self` anywhere.

Reply:
self.num_monte_carlo_points. Yeah, was thinking it could be moved to quadrature.py, but I think that might be scope for a separate PR.